lib/deck.rb

```diff
@@ -0,0 +1,38 @@
+class Deck
+  attr_reader :cards, :high_ranked_cards
+
+  def initialize(cards = [])
```

Comment:
I see that you're adding a default argument here for `cards`, what was the thought process behind this? What stopgaps have been added to `Deck` to be triggered in the scenario where a new `Deck` instance doesn't receive an array of cards?